dlib/cmake_utils/find_blas.cmake

```diff
@@ -287,24 +298,32 @@ elseif(WIN32 AND NOT MINGW)
    if (SIZE_OF_VOID_PTR EQUAL 8)
       set( mkl_search_path
          "C:/Program Files (x86)/IntelSWTools/compilers_and_libraries_*/windows/mkl/lib/intel64" 
+         "C:/Program Files (x86)/IntelSWTools/compilers_and_libraries_*/windows/tbb/lib/intel64/vc14" 
          "C:/Program Files (x86)/IntelSWTools/compilers_and_libraries_*/windows/compiler/lib/intel64" 
-         "C:/Program Files (x86)/IntelSWTools/compilers_and_libraries/windows/compiler/lib/intel64" 
          "C:/Program Files (x86)/IntelSWTools/compilers_and_libraries/windows/mkl/lib/intel64"
+         "C:/Program Files (x86)/IntelSWTools/compilers_and_libraries/windows/tbb/lib/intel64/vc14"
```

Comment:
Is there a way to avoid hard coding versions here too?

Reply:
Not sure about this path, but on my installation is `C:/Program Files (x86)/IntelSWTools/compilers_and_libraries/windows/tbb/lib/intel64/vc_mt`

Reply:
You should have both paths, but you must be compiling with /MT flag, am I right?